### PR TITLE
Stop sending validations as get requests

### PR DESCRIPTION
**Pull request by lmcgrath**
* Supporting both GET and POST requests
* Emitting a deprecation notice for GET requests

### CODE REVIEW

File: config/routes.rb [hunk 1]
```diff
@@ -1,5 +1,5 @@
 Comply::Engine.routes.draw do
-  match '/validations' => 'validations#show', via: [:get]
+  match '/validations' => 'validations#show', via: [:get, :post]
   # allow CORS requests
   match 'validations', via: [:options], to: proc { [200, {}, ['']] }
```
Comment (zvkemp) [inline config/routes.rb]: Is this still valid?

File: lib/comply/version.rb [hunk 2]
```diff
@@ -1,3 +1,3 @@
 module Comply
-  VERSION = '1.8.0'
+  VERSION = '1.8.1'
```
Comment (zvkemp) [inline lib/comply/version.rb]: This is a larger change than I had originally anticipated when we discussed it; feels more like 1.9.0.

Comment (lmcgrath) [inline lib/comply/version.rb]: Bumping the version higher now

File: config/routes.rb (hunk 1 above)
Comment (lmcgrath) [inline config/routes.rb]: What do you mean? Adding multiple request methods?

**Pull request merged**